### DISCUSSION TITLE
arm64 windows build ?

### DISCUSSION THREAD

**Discussion by Jbrimbelibap** [AntiMicroX/antimicrox#420] [Ideas]
a windows arm64 build would be much appreciated

**Comment by pktiuk** [comment]
Why do you need such a build?

**Reply by Jbrimbelibap** [reply]
![16504464230516265845975886258294](https://user-images.githubusercontent.com/90109439/164195789-9b6be55f-651f-423c-8708-55298c43b96c.jpg)

I have an arm64 windows computer. My current solution for the controller is closed source and not arm64 native, i figured i could ask

**Reply by pktiuk** [reply]
That's interesting

Linux already provides arm builds, but there are no plans for adding arm builds for Windows for now (developing and building anything for that system is incredibly annoying).    
If you are familiar with C++ you can try to compile AntiMicroX on your machine (or add building it to our CI).
